packages/features/ee/workflows/pages/workflow.tsx

```diff
@@ -26,6 +26,9 @@ import SkeletonLoader from "../components/SkeletonLoaderEdit";
 import WorkflowDetailsPage from "../components/WorkflowDetailsPage";
 import { getTranslatedText, translateVariablesToEnglish } from "../lib/variableTranslations";
 
+/* eslint-disable @typescript-eslint/no-var-requires */
+const { convert } = require("html-to-text");
```

Comment:
Is this required by the package?

Reply:
I am using `.replace(/<[^>]+>/g, "")` now instead of `convert` from the `html-to-text` package. So I removed the whole package